server/routes/manage-schedules/handlers/activity.ts

```diff
@@ -27,17 +27,31 @@ export default class ActivityRoutes {
       })
     })
 
-    if (!req.session.createJourney) {
-      req.session.createJourney = {}
-      req.session.createJourney.activityId = activity.id
-      req.session.createJourney.category = activity.category
-      req.session.createJourney.name = activity.summary
-      req.session.createJourney.inCell = activity.inCell
-      req.session.createJourney.riskLevel = activity.riskLevel
-      req.session.createJourney.startDate = {
-        day: Number(activity.startDate.substring(8, 10)),
-        month: Number(activity.startDate.substring(5, 7)),
-        year: Number(activity.startDate.substring(0, 4)),
+    req.session.createJourney = {}
```

Comment:
It's a lot of code in a route handler, and OK for now to fix the current issue. I do think its getting ready for a refactor into session-control functions? e.g.  `clearCreateSession()` and `startCreateSession(activity)` and `addToCreateSession(createJourney, activity)` - something along those lines?